server/odc-service/src/main/java/com/oceanbase/odc/service/flow/instance/FlowInstanceConfigurer.java

```diff
@@ -208,10 +211,28 @@ protected FlowInstanceConfigurer next(@NonNull FlowApprovalInstance nextNode,
             userTaskConsumer.accept(builder);
             return builder;
         });
-        targetExecution.next(userTaskBuilder);
+        if (Objects.nonNull(nextNode.getExternalApprovalId())) {
+            String serviceTaskName = "create_external_approval_task_" + nextNode.getId();
```

Comment:
Would it be better to keep the naming format consistent with others?
```Java
FlowNodeType.APPROVAL_TASK.name() + "_external_approval_task_" + nextNode.getId();
```

Reply:
get!